### PR TITLE
[Multiple Cluster] Add cluster to a second region

### PR DESCRIPTION
**Pull request by ferantivero**
Add all required assets to deploy two clusters in two different regions: `eastus2` and `westus2`

Highlights 

- add a second hub
- deploy a second cluster in a different region `westus2`
- modify the resource group name that will home workload-specific services but globally
- improve cleanup section
- some minor fixes (missing exports, clean up, afd, etc.)

### CODE REVIEW

File: 10-frontdoor.md [hunk 1]
```diff
@@ -12,22 +12,21 @@ We will reach two goals:
...
-
+export APPGW_FQDN_BU0001A0042_03=$(az deployment group show --resource-group rg-enterprise-networking-spokes -n spoke-BU0001A0042-03 --query properties.outputs.appGwFqdn.value -o tsv)
+export APPGW_FQDN_BU0001A0042_04=$(az deployment group show --resource-group rg-enterprise-networking-spokes -n spoke-BU0001A0042-04 --query properties.outputs.appGwFqdn.value -o tsv)
 ```
 
 1. Create resource group in order to deploy Azure Front Door
 
 ```bash
-az group create --name rg-global-front-door --location eastus2
+az group create --name rg-bu0001a0042-global --location centralus
```
Comment (v-fearam) [inline 10-frontdoor.md:22]: Please, change the rg name in the cleanup file too.

Comment (ferantivero) [reply 10-frontdoor.md:22]: done

**Review by v-fearam — APPROVED** [review]

File: 02-ca-certificates.md [hunk 2]
```diff
@@ -24,11 +24,11 @@ Following the steps below you will result the certificate needed for Azure Appli
    :warning: We called bicycle3 and bicycle4 each subdomain, but the DNS values could be not available. In that case, you can change the following values.
 
    ```bash
-   CLUSTER_SUBDOMAIN_03=bicycle3
-   CLUSTER_SUBDOMAIN_04=bicycle4
+   export CLUSTER_SUBDOMAIN_03=bicycle3
```
Comment (ferantivero) [inline 02-ca-certificates.md:27]: we had a cumulated commit with all missing exports from [dade9](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/dade95a1f0f54f6be8ed7f9ab84562c1262d69e8).

But it seems in some other [merge](https://github.com/mspnp/aks-secure-baseline/commit/eb4eb480185741e1ab46d2a4580736e5ef1beb64) we could've retired some. 

Anybody knows what is the expected format/style? Personally I'd say, all-or-nothing but I'm open and would love to hear more about this.

cc/ @ckittel

**Review by v-fearam — APPROVED** [review]

File: 02-ca-certificates.md (hunk 2 above)
Comment (ferantivero) [reply 02-ca-certificates.md:27]: remove all the exports for preventing users from scrolling. We are not using the benefits from exporting env vars.

File: 12-cleanup.md [hunk 3]
```diff
@@ -23,12 +23,18 @@ After you are done exploring your deployed [AKS secure baseline cluster](./), yo
    > Because this reference implementation enables soft delete on Key Vault, execute a purge so your next deployment of this implementation doesn't run into a naming conflict.
 
    ```bash
-   az keyvault purge -n $KEYVAULT_NAME
+   az keyvault purge -n $KEYVAULT_NAME_BU0001A0042_03
+   az keyvault purge -n $KEYVAULT_NAME_BU0001A0042_04
    ```
 
-1. If any temporary changes were made to Azure AD or Azure RBAC permissions consider removing those as well.
+1. Delete your Policy Assignments
```
Comment (ferantivero) [inline 12-cleanup.md:30]: Combine with previous text so users can go to portal if they want. 

add some context/comments on what we are doing with the following scripts.

File: 02-ca-certificates.md (hunk 2 above)
Comment (ferantivero) [reply 02-ca-certificates.md]: done | addressed from:
[4b426](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/4b426e834768a086d5bc5c880e991023c2c1f90e)
[55d94](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/55d942653b604a72e380ff2e8728d92148ed5aca)

File: 12-cleanup.md (hunk 3 above)
Comment (ferantivero) [reply 12-cleanup.md]: done | addressed from [540a7](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/540a7de96d222e36124ac9a39d3f48b2003ef247)

**Pull request merged**